api/v1alpha1/gitrepository_types.go

```diff
@@ -54,6 +54,12 @@ type GitRepositorySpec struct {
 	// Verify OpenPGP signature for the commit that HEAD points to.
 	// +optional
 	Verification *GitRepositoryVerification `json:"verify,omitempty"`
+
+	// SourceIgnore overrides the set of excluded patterns with a newline or
```

Comment:
I think we should say here that SourceIgnore format is the same of gitignore.

Reply:
Yeah I just cleaned that up, sorry about the messy patch.